docs/en_US/TrainingService/LocalMode.rst

```diff
@@ -91,7 +93,7 @@ To run an experiment in NNI, you only needed:
 
 ..
 
-   A set of examples can be found in ~/nni/examples after your installation, run ``ls ~/nni/examples/trials`` to see all the trial examples.
+   You can download nni source code and a set of examples can be found in ~/nni/examples, run ``ls ~/nni/examples/trials`` to see all the trial examples.
```

Comment:
`~/nni/examples` is not right. `nni/examples` might be better

Reply:
fix it